models/stp_if_pol.go

```diff
@@ -0,0 +1,100 @@
+package models
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/ciscoecosystem/aci-go-client/container"
+)
+
+const StpIfPolClassName = "stpIfPol"
+
+type SpanningTreeInterfacePolicy struct {
+	BaseAttributes
+	SpanningTreeInterfacePolicyAttributes
+}
+
+type SpanningTreeInterfacePolicyAttributes struct {
+	Name string `json:",omitempty"`
+
+	Annotation string `json:",omitempty"`
```

Comment:
Annotation is a generic attribute. Can we move it to somewhere it will be inherited by everyone?

Reply:
Done as discussed